### ISSUE TITLE
Make settings selector case-insensitive.

### ISSUE THREAD

**Issue by nat-foo** [SupernaturalCow/doc-gen#1]
Currently if you enter "R" or "E" to run or exit the program, it does nothing. Instead you have to enter "r" or "e", which is cumbersome.

**Comment by sachindavra** [comment]
Hi @SupernaturalCow, I am new to opensource and would like to contribute to this project.
Could you explain this in bit detail to undertstand.

**Comment by nat-foo** [comment]
Hi @sachindavra, I am also new to opensource so we can learn together :)

This issue is quite straightforward. Line 295 of `docgen.py` takes the user's input to determine yes/no for certain settings, and line 338 takes the user's input to manage the main menu. For either of these input options, inputting the capital letter variant of the option (e.g. "R" instead of "r") will result in being ignored. A simple fix would be to run .lower() on the resulting string.

Although I think a cleaner option would be to encapsulate `input()` into a custom function that lowers() the input by default, and replace all instances of `input()` with that instead.

**Comment by sachindavra** [comment]
@SupernaturalCow, Thanks for the explanation. I will check on this and update you.

**Comment by sachindavra** [comment]
@SupernaturalCow, How can i run this code out of the box. Do i require to create template file and json file?
if you have template, kindly give it to me. i want to run this code and test it. 

do we have any other communication medium other than github?

**Comment by nat-foo** [comment]
This repo's still very young. We originally created DocGen as part of a proprietary software offering, where we used it for generating APIs for a game engine. We only recently refactored it as a standalone public project to see if anyone else could find a use for it. So we don't have any communication systems in place outside of this repo as of yet, but if it generates enough interest I'll look into setting up something more.

If you'd like to see a sample of how it operates, clone the repo and run `docgen.py` from the root folder. E.g. Run it in VSCode from the root workspace, or run it in Python from cmd line. You'll see it creates a number of example documents in `output/`, which comes from the sample templates in `template/` and sample JSON file in `input/`.
